pages/common/pip-install.md

```diff
@@ -15,6 +15,10 @@
 
 `pip install -r {{requirements.txt}}`
 
+- Install packages from an URL or local file archive (.tar.gz | .whl):
+
+`pip install -f {{url | filename}}`
```

Comment:
```suggestion
`pip install -f {{url|path/to/package_file.tar.gz|whl}}`
```

Reply:
What do you mean by `whl` here?

Reply:
I see. I'm not sure there's really a way to specify that easily. I'd pick the most common filetype and stick with it.

Reply:
I don't know which is more common, however whl is the standard for python packages since its what its designed to do, while tar.gz is just a generic archive  

whl is specific, tar.gz is generic

I'd go with whl

Reply:
Maybe we should just leave it out, since both options are listed in the description.

Reply:
Maybe this is better
```suggestion
`pip install -f {{url|path/to/file}}`
```